packages/cockroach/extra/cockroachdb-change-config.py

```diff
@@ -20,64 +20,70 @@
 logging.basicConfig(format='[%(levelname)s] %(message)s', level='INFO')
 
 
-def set_cluster_version(my_internal_ip: str, version: str) -> None:
-    """
-    Use `cockroach sql` to set the cluster-wide configuration setting
-    version to `version`. This requires that all nodes are running the
-    given version of CockroachDB and are operating successfully.
-
-    Relevant JIRA ticket: https://jira.mesosphere.com/browse/DCOS-19427
-
-    Args:
-        my_internal_ip: The internal IP of the current host.
-        version: The current 'major.minor' version of CockroachDB.
-    """
-    command = [
-        '/opt/mesosphere/active/cockroach/bin/cockroach',
-        'sql',
-        '--insecure',
-        '--host={}'.format(my_internal_ip),
-        '-e',
-        "SET CLUSTER SETTING version = '{}';".format(version),
-        ]
-    config_text = 'version: {}'.format(version)
-    log.info('Set `%s` via command `%s`', config_text, ' '.join(command))
-    subprocess.run(command, input=config_text.encode('ascii'))
-    log.info('Command returned')
-
-
 def set_num_replicas(my_internal_ip: str, num_replicas: int) -> None:
     """
-    Use `cockroach zone set` to set the cluster-wide configuration setting
-    num_replicas to `num_replicas`. This does not matter on a 3-master
-    DC/OS cluster because the CockroachDB default for num_replicas is 3.
-    This however ensures that num_replicas is set to 5 on a 5-master DC/OS
-    cluster. Feed the configuration key/value pair to the `cockroach` program
-    via stdin.
-
+    CockroachDB version 2.1.7 sets the internal system range replication
```

Comment:
As for EE, this can just be 2.1 - but it's only worth changing if re-run is necessary

Reply:
Oops!